index.html

```diff
@@ -1086,9 +1086,25 @@
 
 ## Principles for Vulnerable People
 
-<aside class="issue">
-  This section has not been written.
-</aside>
+Some people, or groups of people, are at greater risk from the asymmetries of power created by the
+concentration of information power ([[?Automating-Inequality]]). [=People=] who for whatever reason
```

Comment:
This can probably be simpler ... maybe

```suggestion
Some people, or groups of people, are more easily coerced into giving up their privacy.
[=People=] who for whatever reason
```